extensions/2.0/Khronos/KHR_nodes_disable/README.md

```diff
@@ -0,0 +1,54 @@
+# KHR_nodes_disable
+
+## Contributors
+
+* Norbert Nopper, UX3D (@UX3DGpuSoftware)
+
+## Status
+
+Draft
+
+## Dependencies
+
+Written against the glTF 2.0 spec.
+
+## Overview
+
+In general, all nodes with its child nodes are traversed and the meshes are rendered every frame. However, there are use cases, where specific meshes should be visible and some should be hidden. During runtime, these nodes are shown or others are hidden. A sample use case are products having specific attachments.  
+Goal is to have all meshes in the glTF and to show/hide specific meshes depending on the product situation.
+  
+If the visible property is `false`, the contents of the node are not visible or used. Main use case is not to render the mesh. By default, the change affects only the specified node. However, if the `recursive` property is set to `true`, all child nodes are affected as well.
```

Comment:
I would suggest to use a direct hiearachy approach and mandate that if a node (parent) is disabled then _all_ children are disabled.
Ie, render traversal stops at the first Node that is disabled.
If this is not the intended behavior I suggest to add a disable property to the Mesh.

Reply:
USD has a similar solution for pruning of visible primitives (geometry)
A node will either have the value inherited or invisible.

Where invisible is always invisible and inherited will compute the visibility value for the parent.
In effect this will turn off all children when a node is hidden (by calling MakeInvisible)
The value is also animatable. 

https://graphics.pixar.com/usd/docs/api/class_usd_geom_imageable.html#aa0dfa061a65be443436288f7406ba3db

Reply:
Think we should just make it the same.